test/words/test/TestINodeReferenceList.java

```diff
@@ -0,0 +1,67 @@
+package words.test;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import words.ast.*;
+import words.environment.*;
+import words.exceptions.*;
+
+public class TestINodeReferenceList extends TestINode {
+	
+	@Test
+	public void testSizeZero() throws WordsRuntimeException {
+		INodeReferenceList refList = new INodeReferenceList();
+		ASTValue nothingValue = refList.eval(environment);
+		
+		assertEquals(nothingValue.type, ASTValue.ValueType.NOTHING);
+	}
+	
+	@Test
+	public void testSizeOne() throws WordsRuntimeException {
+		WordsObject alexObject = environment.createObject("Alex", "thing", new WordsPosition(0, 0));
+		LNodeReference alexRef = new LNodeReference("Alex's");
+		INodeReferenceList alexRefList = new INodeReferenceList(alexRef);
+		
+		ASTValue alexValue = alexRefList.eval(environment);
+		assertEquals(alexValue.objValue, alexObject);
+	}
+	
+	@Test
+	public void testSizeTwo() throws WordsRuntimeException {
```

Comment:
For thoroughness, could we make this a test of three (or even four) instead of just two?  I'd feel more confident that that will capture the general case, whereas effecitvely only captures one iteration of the reference loop and therefore doesn't test the iteration.